### PR TITLE
New package: pam-mount-2.15

### PR DESCRIPTION
**Pull request by Vaelatern**
Issue #3119

### CODE REVIEW

File: srcpkgs/pam-mount/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+maintainer="Toyam Cox <Vaelatern@gmail.com>"
+license="GPL-3"
+homepage="http://pam-mount.sourceforge.net/"
+distfiles="${SOURCEFORGE_SITE}/pam-mount/pam_mount/${version}/pam_mount-${version}.tar.xz"
+checksum=a54efb4a8e8dd52c77b15cd36d9354b4733bb9b4cae75083801c18e67fc12a24
+
+libpam-mount_package() {
+	short_desc+=" - runtime library"
+	pkg_install() {
+		vmove usr/lib/*.so*
```
Comment (xtraeme) [inline srcpkgs/pam-mount/template]: `vmove usr/lib/*.so.*`

File: srcpkgs/pam-mount/template [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		vmove usr/lib/security/*.so*
+	}
+}
+
+libpam-mount-devel_package() {
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include/*.h
+	}
```
Comment (xtraeme) [inline srcpkgs/pam-mount/template]: `vmove usr/lib/*.so`

File: srcpkgs/pam-mount/template [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+libpam-mount_package() {
+	short_desc+=" - runtime library"
+	pkg_install() {
+		vmove usr/lib/*.so*
+		vmove usr/lib/security/*.so*
+	}
+}
+
+libpam-mount-devel_package() {
+	short_desc+=" - development files"
```
Comment (xtraeme) [inline srcpkgs/pam-mount/template]: `depends="libpam-mount>=${version}_${revision}"`

Comment (Vaelatern) [inline srcpkgs/pam-mount/template]: This change has been made.

**Comment by xtraeme** [comment]
Please rebase to get this merged.

**Pull request merged**